commands.sh

```diff
@@ -64,10 +64,10 @@ alias tdrs-run-jest='tdrs-npm-run test'
 alias tdrs-run-jest-cov='tdrs-npm-run test:cov'
 
 # run any new migrations for django backend
-alias tdrs-run-migrations='tdrs-compose-backend run web sh -c "python manage.py migrate"'
+alias tdrs-run-migrations='tdrs-compose-backend run web python manage.py migrate'
 
 # Generate new migrations from changes to models for django backend
-alias tdrs-run-migrations='tdrs-compose-backend run web sh -c "python manage.py makemigrations"'
+alias tdrs-make-migrations='tdrs-compose-backend run --rm web python manage.py makemigrations'
```

Comment:
For these commands to work in my environment, needed to make this change to a direct invocation of python rather than bash calling python.